gke/config.yaml

```diff
@@ -0,0 +1,21 @@
+# Copyright 2020 Google LLC
```

Comment:
Is this file a template? Should we rename the file to indicate that (I think we have a few already), and make it clear in the instructions to create a new copy of this file?

Reply:
Made this a template file and updated instructions.